tests/issues/100/sap-preconfigure-issue-100-assert.yml

```diff
@@ -0,0 +1,50 @@
+- hosts: all
+  tasks:
+  - name: Get the hard limit of nofile for group sapsys
+    command: awk '/@sapsys/&&/nofile/&&/hard/{print $NF}' /etc/security/limits.d/99-sap.conf
+    register: awk_nofile_hard_result
```

Comment:
is it possible that `/etc/security/limits.d/99-sap.conf` does not exist?  If so, you might want to `ignore_errors: true`, or maybe it is ok to let this task fail?

Reply:
Thanks for this, will add a stat task before this one.